types/react-native-video/index.d.ts

```diff
@@ -0,0 +1,77 @@
+// Type definitions for react-native-swiper 1.0
+// Project: https://github.com/react-native-community/react-native-video
+// Definitions by: HuHuanming <https://github.com/huhuanming>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+import * as React from 'react';
+import {
+    ViewProperties
+} from 'react-native';
+
+export interface VideoProperties extends ViewProperties {
+    /* Native only */
+    src?: any;
```

Comment:
Sure, but that seems to be more because this package doesn’t have any real JSDoc and that comment is meant as an internal detail, the user of the package doesn’t really need to know about this.

I definitely think that taking the comments from the usage example will make the typings much nicer to use.